content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -310,6 +310,36 @@ assuming that the number of replicas is not also changed).
 A ReplicaSet can be easily scaled up or down by simply updating the `.spec.replicas` field. The ReplicaSet controller
 ensures that a desired number of Pods with a matching label selector are available and operational.
 
+### Pod Deletion Cost 
```

Comment:
nit:
```suggestion
### Pod Deletion Cost 
```

Reply:
These are exactly the same 😂 
Perhaps you were referring to `Pod deletion cost`?

Reply:
Ah yep oops

Reply:
done.